### DISCUSSION TITLE
Deprecate/replace AuthRequest.validate()

### DISCUSSION THREAD

**Discussion by eurekaP** [ProLord007/Auth_Lib#1] [Q&A]
How can I deprecate or replace AuthRequest.validate() on this library?

**Comment by ProLord007** [comment]
There's 2 ways to validate requests using AuthRequest:
```
authRequest.validate();
authRequest.validateUser();
```
validate() returns the session, and validateUser() returns the session and user. It isn't really useful to have both, and if you're using SvelteKit, using both in a single load function flow can lead to 2 database calls (depending on the other).

I think we should replace validate() with the current validateUser(), so it returns both user and session.
